standard/classes.md

```diff
@@ -2222,7 +2245,7 @@ A parameter declared with a `params` modifier is a parameter array. If a formal
 
 > *Example*: The types `string[]` and `string[][]` can be used as the type of a parameter array, but the type `string[,]` can not. *end example*
 
-It is not possible to combine the `params` modifier with the modifiers `ref` and `out`.
+It is not possible to combine the `params` modifier with the modifiers `in`, `out`, and `ref`.
```

Comment:
That’s been there a very long time, no idea why this form of language was used. Line 1971 above:

> It is a compile-time error to have both `ref` and a *return_type* of `void`.

suggests the wording:

```suggestion
It is a compile-time error to combine the `params` modifier with the modifiers `in`, `out`, or `ref`.
```

(I changed “and” to “or” as it might be read that `params in out ref` is invalid?)

Note however that the grammar does not allow `params` and one of the *parameter_mode_modifier* anyway so an alternative suggestion is:

```suggestion
```

Or make it a note:

```suggestion
*Note*: It is not possible to combine the `params` modifier with the modifiers `in`, `out`, or `ref`. *end note*
```

Choice, choices, take your pick! 🙂

Reply:
I prefer the third optin.